hack/build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+function lifecycle_image_build() {
+    image=$1
+    go run hack/lifecycle/main.go --tag=${image}
+}
+
+function generate_kbld_config_pack() {
```

Comment:
There's something weird about this when you try to rebuild the image again. For some reason it results in a repo digest mismatch. I wonder if kbld is adding something that's causing the build to not be reproducible?

Reply:
This was due to my env being polluted, nuking the registry and docker pruning fixed the problem